express-api/src/controllers/admin/roles/rolesController.ts

```diff
@@ -35,8 +41,8 @@ export const addRole = async (req: Request, res: Response) => {
       }]
    */
 
-  // TODO: Replace stub response with controller logic
-  return stubResponse(res);
+  const role = await rolesServices.addRole(req.body);
+  return res.status(201).send(role);
```

Comment:
I skipped out on some of that logic because I think we need to decide on how we should verify these request bodies in a general sense. Maybe I should just see how well [this](https://orkhan.gitbook.io/typeorm/docs/validation) works and if it's fine enough we'll just stick to it? If not we can just stick to zod.